resourcet/Data/Acquire/Internal.hs

```diff
@@ -25,8 +27,21 @@ import qualified Control.Monad.Catch as C ()
 -- @since 1.1.2
 data ReleaseType = ReleaseEarly
                  | ReleaseNormal
-                 | ReleaseException
-    deriving (Show, Read, Eq, Ord, Enum, Bounded, Typeable)
+                 | ReleaseException' E.SomeException
```

Comment:
Gonna take issue with this name - the `'` feels unnecessary when a word can be more descriptive, and this isn't, like, "a strict `ReleaseException`", so the most common applicable mnemonic doesn't hold.

Reply:
No strong opinions.

Reply:
Done.